src/borg/crypto/low_level.pyx

```diff
@@ -673,7 +676,7 @@ cdef class _CHACHA_BASE(_AEAD_BASE):
 cdef class AES256_OCB(_AES_BASE):
     @classmethod
     def requirements_check(cls):
-        if OPENSSL_VERSION_NUMBER < 0x10100000:
+        if OPENSSL_VERSION_NUMBER < 0x10100000 or LIBRESSL_VERSION_NUMBER:
```

Comment:
maybe we can just use the openssl10 global?

Reply:
Guess it makes sense.